src/tests/ftest/recovery/pass_0.yaml

```diff
@@ -0,0 +1,17 @@
+hosts:
+  test_servers: 2
+
+timeout: 180
+
+server_config:
+  name: daos_server
+  engines_per_host: 1
+  engines:
+    0:
+      targets: 1
+      nr_xs_helpers: 0
```

Comment:
Any reason why nr_xs_helpers is set to 0. Can we set to 1... Also, why targets: 1... Why not use the default targets?